File: Misc/cafea.cpp
#include<iostream>
#include<fstream>
#include<windows.h>
#include<math.h>
using namespace std;
int main()
{
  int x,optiune;
  ifstream f("fis1.in");
  f>>x;
  if(x != 5)
    {
      x++;
      f.close();
      ofstream rez("fis1.in");
      rez<<x;
    }
  else
    {
      cout<<"Trebuie sa pui apa in espresor.";
      ofstream rez("fis1.in");
      x = 0;
      rez<<x;
      rez.close();
      return 0;
    }
  cout<<"Cine vrea sa-si faca cafea? \n1. Bobo. \n2. Mami. \n3. Tati. \nOptiune: ";
  cin>>optiune;
  if(optiune == 1)
    {
      cout<<"Cafeaua lui Bobo va fi gata in 3 minute.\n";
    Sleep(3000);
      cout<<"Cafeaua lui Bobo este gata.";
    }
  else if(optiune == 2)
    {
      cout<<"Cafeaua lui Mami va fi gata in 3 minute.\n";
    Sleep(3000);
      cout<<"Cafeaua lui Mami este gata.";
    }
  else if(optiune == 3)
    {
      cout<<"Cafeaua lui Tati va fi gata in 3 minute.\n";
    Sleep(3000);
      cout<<"Cafeaua lui Tati este gata.";
    }
}

File: Python/cafea.py
import time
import random
from sys import exit

apa = random.randrange(0,2)
if apa == 0:
    print("Trebuie sa pui apa in espresor")
    exit()
elif apa == 1:
    print("Nu trebuie sa pui apa in espresor")
print("Cine vrea sa-si faca cafea? \n1.Bobo. \n2.Mami. \n3.Tati.")

cafea = int(input("Optiune: "))

if cafea == 1:
    print("Cafeaua pentru Bobo va fi gata in 3 minute.")
    time.sleep(3)
    print("Cafeaua lui Bobo este gata.")

if cafea == 2:
    print("Cafeaua pentru mami va fi gata in 3 minute.")
    time.sleep(3)
    print("Cafeaua lui mami este gata.")

if cafea == 3:
    print("Cafeaua pentru tati va fi gata in 3 minute.")
    time.sleep(3)
    print("Cafeaua lui tati este gata.")
